### PR TITLE
feat: project description (DEV-1452)

### PR DESCRIPTION
**Pull request by mdelez**
resolves DEV-1452

### CODE REVIEW

**Comment by mdelez** [comment]
for the reviewers, I spoke with Rebecca about the design and she's fine with how it looks but all the functionality is there

File: src/app/project/description/description.component.ts [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+    sysAdmin = false;
+    projectAdmin = false;
+
+    // project uuid; as identifier in project cache service
+    projectUuid: string;
+
+    // project data
+    project: ReadProject;
+
+    color = 'primary';
```
Comment (Vijeinath) [inline src/app/project/description/description.component.ts:44]: This member is not used in the file or template

File: src/app/project/description/description.component.ts [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+            this.keywords.push(value.trim());
+        }
+
+        // reset the input value
+        if (input) {
+            input.value = '';
+        }
+    }
+
+    removeKeyword(keyword: any): void {
```
Comment (Vijeinath) [inline src/app/project/description/description.component.ts:216]: _any_ should be replaced with _string_

File: src/app/project/description/description.component.ts [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+        projectData.longname = this.form.value.longname;
+        projectData.status = true;
+
+        let i = 0;
+        for (const d of this.description) {
+            projectData.description[i] = new StringLiteral();
+            projectData.description[i].language = d.language;
+            projectData.description[i].value = d.value;
+            i++;
+        }
```
Comment (Vijeinath) [inline src/app/project/description/description.component.ts:253]: To save two lines you could use:
```
for (let i = 0; i < this.description.length; i++) {
  xxx
}
```

**Review by Vijeinath — COMMENTED** [review]
Just small things

File: src/app/project/description/description.component.ts (hunk 2 above)
Comment (mdelez) [reply src/app/project/description/description.component.ts]: done in [ab03e17](https://github.com/dasch-swiss/dsp-app/pull/928/commits/ab03e177a0cf0cce96f985c7abb728fd33929a5c)

File: src/app/project/description/description.component.ts (hunk 1 above)
Comment (mdelez) [reply src/app/project/description/description.component.ts]: done in [ab03e17](https://github.com/dasch-swiss/dsp-app/pull/928/commits/ab03e177a0cf0cce96f985c7abb728fd33929a5c)

**Review by Vijeinath — APPROVED** [review]
Good

**Pull request merged**